### PR TITLE
Card-Animation

### PR DESCRIPTION
**Pull request by ChitvanRamani22**

### CODE REVIEW

**Comment by Namanl2001** [comment]
15k lines of change? Have you not followed instructions to setup the project? We use yarn to install the dependencies but it seem like that you have used npm install instead of yarn install.
To correct this, delete package-lock.json in the backend folder, run yarn install in the same folder, commit your changes and push them again in this branch only.

File: src/App.css [hunk 1]
```diff
@@ -31,3 +31,11 @@ html {
     margin: auto;
   }
 }
+.card{
+  transition: transform .8s ease;
+}
+.card:hover{
+  transition: .8s;
+  transform: scale(1.14);
+  box-shadow: 4px 4px #D3D3D3;
```
Comment (Namanl2001) [inline src/App.css:40]: transform: scale(1.1);
box-shadow: 6px 6px #D3D3D3;

File: package.json [hunk 2]
```diff
@@ -9,6 +9,7 @@
     "axios": ">=0.21.1",
     "bootstrap-4-react": "0.0.59",
     "bootstrap4": "0.0.1-security",
+    "concurrently": "^6.0.0",
```
Comment (Namanl2001) [inline package.json:12]: Remove this, and run `yarn install` from the root of this repo.

File: backend/server.js [hunk 3]
```diff
@@ -3,7 +3,9 @@ const cors = require("cors");
 const mongoose = require("mongoose");
 const bodyParser = require("body-parser");
 const path = require("path");
-const { username, pass, myCluster } = require('./config');
+//const { username, pass, myCluster } = require('./config');
+const config = require('./config.js');
+const { username, pass,myCluster  } = config;
```
Comment (Namanl2001) [inline backend/server.js:8]: Not needed. uncomment line no. 6 and remove line no. 7 & 8

File: backend/routes/users.js [hunk 4]
```diff
@@ -1,7 +1,9 @@
 const router = require("express").Router();
 var nodemailer = require("nodemailer");
 let User = require("../models/user.model");
-const { pass } = require('./config');
+//const { pass } = require('./config');
+const config = require('../config');
+const {pass}=config;
```
Comment (Namanl2001) [inline backend/routes/users.js:6]: `const { pass } = require('../config'); ` should work.

**Review by Namanl2001 — CHANGES_REQUESTED** [review]
Good work! Some changes are needed.

File: backend/routes/users.js [hunk 5]
```diff
@@ -14,7 +14,6 @@ router.route("/:email").get((req, res) => {
     .then((users) => res.json(users))
     .catch((err) => res.status(400).json("Error: " + err));
 });
-
```
Comment (Namanl2001) [inline backend/routes/users.js:17]: seems like a non intentional change.

**Review by Namanl2001 — CHANGES_REQUESTED** [review]
small nit

**Review by Namanl2001 — APPROVED** [review]
LGTM! Thanks.

**Comment by ChitvanRamani22** [comment]
Thanks a lot sir.
Do add this to my gssoc contribution.

**Comment by ChitvanRamani22** [comment]
Sir just to clarify that i am a girl.
Thanks

**Pull request merged**